server/datastore/mysql/hosts.go

```diff
@@ -1648,11 +1652,63 @@ WHERE
 }
 
 func (ds *Datastore) UpdateOSVersions(ctx context.Context) error {
+	counts, err := ds.gatherOSCountsDB(ctx)
+	if err != nil {
+		return ctxerr.Wrapf(ctx, err, "gathering counts stats for os versions")
+	}
+
+	teams, err := ds.TeamsSummary(ctx)
+	if err != nil {
+		return ctxerr.Wrapf(ctx, err, "getting team summary")
+	}
+
+	countsByTeamId := make(map[uint]*json.RawMessage)
+	for _, c := range *counts {
+		countsByTeamId[c.TeamID] = c.OSVersions
+	}
+
+	for _, t := range teams {
```

Comment:
Please add a comment that we need to do this because `json_value` is `NOT NULL` and we would like to explicitly set an empty array for the empty teams (guessing that's why)